Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM alpine:3.12
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 ARG BUILD_DATE
 ARG VCS_REF
+ARG MARIADB_VERSION="10.4.18-r0"
```

Comment:
Can you skip this for now? I will do bigger changes here to push multiple versions.